en/frames_rover/traxxas_stampede.md

```diff
@@ -0,0 +1,138 @@
+
+
+## Configuration
+
+Rovers are configured using *QGroundControl* in the same way as any other vehicle.
+
+The main rover-specific configuration is setting the correct frame:
+1. Switch to the [Basic Configuration](../config/README.md) section in *QGroundControl*
+1. Select the [Airframe](../config/airframe.md) tab. 
+1. Scroll down the list to find the **Rover** icon. 
+1. Choose **Traxxas stampede vxl 2wd** from the drop down list.
+
+![Select Airframe](../../images/qgc/setup/airframe_px4_rover_traxxas_stampede_vxl_2wd.jpg)
+
+
+## Usage
+
+At the current time, PX4 only supports MISSION and MANUAL modes when a RC remote is connected. To use the mission mode, first upload a new mission to the vehicle with QGC. Then, BEFORE ARMING, select `MISSION` and then arm. 
+
+> **Warning** It is **_VERY IMPORTANT_** to do a mission composed **_ONLY_** of normal waypoints (i.e. **_NO TAKEOFF WAYPOINTS_**) and it is crucial to **_SET THE WAYPOINT HEIGHT OF EVERY WAYPOINT TO 0_** for a correct execution. Failing to do so will cause the rover to continuously spin around a waypoint.
+
+
+A correct mission setup looks as follows
+
+![mission](../../assets/airframes/rover/traxxas_stampede_vxl/correct_mission.jpg)
+ 
+Since at the current state of development there is no obstacle avoidance, it is highly recommended to test missions in the simulator first to get acquainted with the software usage.
+
+
+<!-- 
+
+## Parameters
+
+The current software provides an attitude controller and a position controller. These build on top of the fixed wing framework to move an Ackerman steered vehicle following GPS coordinates (real or fake). In order to this, 2 PIDs were implemented, one for the steering and the other for the speed.
+
+
+These can be started by setting the following general parameters, which can be set with *QGroundControl* (instructions [here](../SetupView/Parameters.md)):
+* SYS_AUTOSTART 50002
+* MAV_TYPE 10
+* MIS_LTRMIN_ALT 0.01
+* MIS_TAKEOFF_ALT 0.01
+
+This allows to start the Pixhawk configured for the Traxxas Stampede VXL. Then, the steering controller can be tuned with these parameters:
+* GND_WR_D 0.1
+* GND_WR_I 0.01
+* GND_WR_IMAX 0.1
+* GND_WR_P 1.5
+
+The closed loop speed control can be enabled by setting 
+* GND_SP_CTRL_MODE 1
+
+which uses the second PID to control the speed with the following parameters:
+* GND_SPEED_D 5
+* GND_SPEED_I 0.01
+* GND_SPEED_IMAX 0.5
+* GND_SPEED_P 2.0
+* GND_SPEED_THR_SC 0.1
+* GND_SPEED_TRIM 4
+
+If the closed loop is enabled the speed controller will take the GND_SPEED_TRIM as a target speed and will attempt to track it. If, on the other hand, GND_SP_CTRL_MODE is set to 0 then the speed controller will turn to a safe open loop setting of the throttle, controlled by these parameters:
+* GND_THR_CRUISE 0.1
+* GND_THR_IDLE 0
+* GND_THR_MAX 0.5
+* GND_THR_MIN 0
+
+Then, general navigation parameters can be tuned for a better usage: 
+* GND_L1_DIST 3
+* NAV_ACC_RAD 0.5
+-->
+
+<!--
```

Comment:
I am proposing to delete this section (not appropriate in user guide), but before I do I want to make sure that the devguide has appropriate mention of rover building and simulation.